Public/Src/Tools/Execution.Analyzer/Analyzers.Core/ProcessRunScriptAnalyzer.cs

```diff
@@ -251,20 +272,22 @@ private IEnumerable<AbsolutePath> GetOutputDirectories(Process pip)
 
         private void DeleteOutputs(StreamWriter writer, Process pip)
         {
-            writer.WriteLine(":::: Delete Outputs");
+            writer.WriteLine($"{CommentPrefix} Delete Outputs");
 
-            writer.WriteLine(":: Delete Files");
+            writer.WriteLine($"{CommentPrefix} Delete Files");
             foreach (var fileOutput in pip.FileOutputs)
             {
-                writer.WriteLine("IF EXIST \"{0}\" del \"{0}\"", fileOutput.Path.ToString(PathTable));
+                var path = fileOutput.Path.ToString(PathTable);
+                writer.WriteLine(ScriptIfPathExistsConditionWithCommand(path, $"{FileDeleteOperation} \"{path}\""));
             }
             writer.WriteLine();
 
-            writer.WriteLine(":: Delete Directories");
+            writer.WriteLine($"{CommentPrefix} Delete Directories");
             foreach (var directory in GetOutputDirectories(pip))

```

Comment:
>oreach (var directory in GetOutputDirectories(pip)) [](start = 13, length = 51)

This comes from a time before shared opaque directories. It's a bit sketchy now since there may actually be inputs in these directories. Embedding more precise file deletion logic in this script is overkill. I'm not sure what the best way to go is. Maybe some sort of "proceed with caution" warning is in order. What do you think?

Reply:
I've added a warning to this is could be an issue.